src/components/v5/common/ActionSidebar/partials/forms/UnlockTokenForm/hooks.ts

```diff
@@ -37,18 +37,12 @@ export const useUnlockToken = (
     transform: useCallback(
       pipe(
```

Comment:
Here pipe makes some sense, as you're both adding a key/id and mapping the payload